lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -312,6 +314,13 @@ def get(self, request, *args, **kwargs):
                         seq_data['type'] != 'sequential'
                     )
                 ] if 'children' in chapter_data else []
+        
+        if not request.user.is_anonymous:
+            user_grade = CourseGradeFactory().read(request.user, course)
+            if user_grade:
```

Comment:
What if the user isn't anonymous, and also doesn't have a grade?

Reply:
whoops, good point. Changed it so `user_has_passing_grade` defaults to false.

Reply:
(Similar thought but you fixed this while I was typing it!)